src/rapids_singlecell/preprocessing/_pca.py

```diff
@@ -137,51 +139,79 @@ def pca(
             n_comps = min_dim - 1
         else:
             n_comps = 50
+    if isinstance(X, DaskArray):
+        if chunked:
+            raise ValueError(
+                "Dask arrays are not supported for chunked PCA computation."
+            )
+        _check_gpu_X(X, allow_dask=True)
+        if not zero_center:
+            raise ValueError("Dask arrays do not support non-zero centered PCA.")
+        if isinstance(X._meta, cp.ndarray):
+            from cuml.dask.decomposition import PCA
+
+            if svd_solver == "auto":
+                svd_solver = "jacobi"
+            pca_func = PCA(n_components=n_comps, svd_solver=svd_solver, whiten=False)
+            X_pca = pca_func.fit_transform(X)
+            X_pca = X_pca.compute_chunk_sizes()
```

Comment:
Why do we need to compute chunk sizes here?

Reply:
I added a comment for PCA `X_pca = X_pca.compute_chunk_sizes()`.

For the copying in HVG. Since I use calculate_qc for to find where to subset. I wanted to be sure that the original data doesn't get overwritten. I addition to that the dask gpu views are sometimes a bit weird. So copy makes this more solid in general. That applies most to slicing against the minor axis.

Reply:
I also dont want dask to be a dependency because cuml handels that. I dont want get into problems there.